app/views/posts/index.html.erb

```diff
@@ -1 +1,16 @@
 <h1>Here is a list of posts for a given user</h1>
+<div class='container'>
+<% @posts.each_with_index do |post, index| %>
+  <% link = "posts/#{post.id}" %>
+  <a href=<%= link %>>
+    <div class='user-posts'>
```

Comment:
- [ ] There is an error raised here, Please fix that.

![image](https://user-images.githubusercontent.com/68796232/223889004-c0d6421f-ff67-48f3-9de0-c40c074d2fd1.png)